typescript/src/client/sse.ts

```diff
@@ -50,8 +50,11 @@ export async function createEventSource({
         for await (const message of stream) {
           yield message;
         }
-      } catch (err) {
-        throw new SSEError((err as Error).message, response, { cause: err });
+      } catch (e) {
+        const err = e as Error;
+        if (err.name !== 'AbortError') {
+          throw new SSEError(err.message, response, { cause: err });
+        }
```

Comment:
We should re-throw the abort error here. The `runStream` should not finish without the exception.